### ISSUE TITLE
Deprecated filters: event_exists? broken (sensu 0.26.3)

### ISSUE THREAD

**Issue by Hydrochoerus** [sensu-plugins/sensu-plugin#156]
At least filter_dependencies depends on event_exists function (sensu-handler.rb), which is broken. I would appreciate if this was fixed, as there is no replacement for the deprecated dependency filtering yet. 

```ruby
def event_exists?(client, check)
      api_request(:GET, '/event/' + client + '/' + check).code == '200'
end
```

With the new API should be (event => events):

```ruby
def event_exists?(client, check)
      api_request(:GET, '/events/' + client + '/' + check).code == '200'
end
```

I am running sensu 0.26.3.

**Comment by cwjohnston** [comment]
Hi @Hydrochoerus, thanks for opening this issue. I've marked it as a bug and we can probably publish a fix for it shortly. For what it's worth, it looks like this has been broken since 0.25.0 (circa 2016-06-13) [due to the removal of singular form endpoints in Sensu's API](https://github.com/sensu/sensu/blob/master/CHANGELOG.md#important).

**Comment by cwjohnston** [comment]
Closed by #157
